assets/build/config.js

```diff
@@ -26,6 +26,7 @@ const config = mergeWithConcat({
     watcher: !!argv.watch,
   },
   watch: [],
+  browsers: ['last 2 versions', 'android 4', 'opera 12'],
```

Comment:
Can you make this line:
```json
  browsers: []
```

And add those browser versions to `config.json` instead?

The reason is because arrays are concatenated by `mergeWithConcat()`.